src/entity/presentationDefinition/presentationDefinitionWrapperEntity.ts

```diff
@@ -10,20 +13,14 @@ export class PresentationDefinitionWrapperEntity {
   // @ts-ignore
   _id: ObjectID;
 
-  @Column()
-  presentation_definition?: PresentationDefinition;
-
   @Column()
   // @ts-ignore
-  id: string;
+  presentation_definition: PresentationDefinition;
 
   @Column()
   // @ts-ignore
-  comment: string;
```

Comment:
why did you remove the comment property?

Reply:
Reverted the changes.